exts/cesium.powertools/cesium/powertools/powertools_window.py

```diff
@@ -50,3 +51,6 @@ def _build_fn(self):
         with ui.VStack(spacing=4):
             for action in self._actions:
                 action.button()
+
+    def _open_georeference_helper(self):
+        CesiumGeorefHelperWindow(width=250, height=550)
```

Comment:
Move this to a public static method on `CesiumGeorefHelperWindow`.

Reply:
Done - new method is called create_window